task.go

```diff
@@ -116,6 +116,8 @@ func (r *marathonClient) KillApplicationTasks(id string, opts *KillApplicationTa
 //	opts:		KillTaskOpts request payload
 func (r *marathonClient) KillTask(taskID string, opts *KillTaskOpts) (*Task, error) {
 	appName := taskID[0:strings.LastIndex(taskID, ".")]
+	appName = strings.Replace(appName, "_", "/", -1)
```

Comment:
Aren't we yet missing a test that passes an underscored app ID into `KillTask` and makes sure we're seeing the proper replacement?